inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,328 @@
+## design study. You will then need to identify your weights variable. 
+## You can also create several study designs, e.g. if you would like to have
+## an overall design and a stratified design (e.g. by sex or by region). 
+##
+## Checklist for study design --------------------------------------------------
+## 
+## - [ ] Make sure you have created all necessary variables in earlier code chunks!!
+## - [ ] Decide on your study design (cluster or simple random)
+## - [ ] If doing cluster:
+##          - [ ] place your cluster ID variable in the ids option 
+##                (e.g. householdID - "fact_0_id" not availble in dat dict)
+##          - [ ] specify your weight variable (created previously)
+##                (set to NULL if want no weights)
+## - [ ] If doing simple random: 
+##          - [ ] set ids equals 1 to have no clustering 
+##          - [ ] specify your weight variable (created previously)
+##                (set to NULL if want no weights)
+## - [ ] Create additional study designs based on strata 
+##      (make sure that your weight variable is based on appropriately stratified
+##        population data)
+
+
+# simple random sample (using srvyr package)
+survey_design <- study_data_cleaned %>% 
+  as_survey_design(ids = 1, # 1 for no cluster ids 
+                   weights = weight # weight variable created above 
+                   )
+
+
+# simple random sample stratified by sex 
+survey_design_sex <- study_data_cleaned %>% 
+  as_survey_design(ids = 1, # 1 for no cluster ids 
+                   weights = weight, # weight variable created above 
+                   strata = sex
+                   )
 ```
 
 
-## Sampling
 
-### Random sampling
+
+
+# Introduction
+
+# Methods
+
+## Study Design 
+
+## Sample Size and Sampling methods
+
+### Sample Size
+
+### Sampling
+
+#### Random sampling
 
 ```{r}
 # Insert some code to generate a random number sequence
 ```
 
-### Cluster sampling
+#### Cluster sampling
 
 ```{r}
 # Insert some example code to do cluster sampling
 ```
 
-# Study Findings
+## Data collection
 
+## Data analysis
+<!-- TODO: add paragraph about R/sitrep -->
+
+# Results
 
 ```{r}
-# we first need to create a survery design that reflects the study
-suvery_design <- svydesign(
-  ids = ~1, # no cluster ids
-  weights = NULL, #no sampling weights in this example
-  data = study_data_cleaned
-)
+# TODO: Arrivals and departures of included household members during the recall period;
 ```
 
 
-## Demographics
+## Demographic information
 
 ```{r}
-plot_age_pyramid(study_data_cleaned)
+# add descriptive table with unweighted counts for different stuff
 ```
 
-## Morbitity
-
-```{r}
-# symptoms start with sym_
+TODO: add some sample text with inline R
+
+Age distribution of current household population by five year age groups, dependency age groups, and by child and adult populations, by gender.
+```{r describe_by_age_group_and_sex}
+descriptive(study_data_cleaned, 
```

Comment:
change to use proptotal = TRUE

Reply:
added it.
in the template on google drive it was proportionate by gender, thats why i did it here as well.